### ISSUE TITLE
More styled buttons

### ISSUE THREAD

**Issue by NehemiahK** [NehemiahK/Another-React-Lib#11]
We already have some buttons but we can always use more, maybe some that are good for call to actions or have different hover effects, maybe one where the text changes on hover.

**Comment by 0c0de** [comment]
Can I take it?

**Comment by NehemiahK** [comment]
Sure thing. Want to create an issue for the button you plan to make?

**Comment by nataliepls** [comment]
Hey can I also add some buttons? :)

**Comment by NehemiahK** [comment]
@nataliepls yea, can you open a new issue for the button you want to make?

**Comment by nataliepls** [comment]
@NehemiahK I don't know how but I will try :D It's my first time with github

**Comment by NehemiahK** [comment]
@nataliepls do you have an option to create 'new issue' on this page?

**Comment by nataliepls** [comment]
@NehemiahK yes, i have. I will try to make it look right but feel free to correct me, if you see something is wrong :)

**Comment by RuchiraRuwantha** [comment]
Hey, can I also create some buttons?

**Comment by NehemiahK** [comment]
@RuchiraRuwantha of course just create an issue with what buttons you'll be making

**Comment by RuchiraRuwantha** [comment]
@NehemiahK do you have any suggestions?

**Comment by NehemiahK** [comment]
Maybe a sticky button that's on the side of the screen, when you click it it takes you back to the the top. and it could come in different themes?

**Comment by RuchiraRuwantha** [comment]
> 
> 
> Maybe a sticky button that's on the side of the screen, when you click it it takes you back to the the top. and it could come in different themes?

Ok. I'll look into it.

**Comment by karan171996** [comment]
@NehemiahK  Hey, can I make the sticky buttons as you want??

**Comment by NehemiahK** [comment]
Sticky button is being worked on. But there’s other buttons I’m sure we could use

**Comment by benfluleck** [comment]
I will come back to this after one of the other buttons are done I think. Your Button component should have a type prop to render different button styles. This keeps the logic localized and more manageable

**Comment by NehemiahK** [comment]
@benfluleck i think you're correct
